src/main.py

```diff
@@ -1,16 +1,28 @@
-# This is a sample Python script.
+import unittest
+
+
+class ReturnClass:
+    def return_number(self):
+        return 897
+back = ReturnClass()
+resolution = back.return_number()
+print(f"number = {resolution}")
+
+class Test_ReturnClass(unittest.TestCase):
+    def test_return_number(self):
+        back = ReturnClass()
+        test_resolution = back.return_number()
+        self.assertEqual(test_resolution, 897)
+
+if __name__ == "__main__":
+    unittest.main()
```

Comment:
This should be in different file. Make another directory next to src called `tests` in that tests directory make directory `unit_tests` and then in this unit_tests directory create a file with this test. 

From what i know you don't need 
```
if __name__ == "__main__":
    unittest.main()
```
I will tell you how to call them in PyCharm without this